src/modules/universal-discovery/components/search/search.component.js

```diff
@@ -86,6 +88,33 @@ export default class SearchComponent extends Component {
         }
     }
 
+    /**
+     * Renders the language filter
+     *
+     * @method renderLanguageFilter
+     */
+    renderLanguageFilter() {
+        const selectAttrs = {
+            className: 'form-control',
+            ref: this.setSearchLanguageRef
+        };
+
+        const { languages } = this.props;
+
+        const options = languages.priority.map((value) => {
+            const language = languages.mappings[value];
+            return <option value={language.languageCode}>{language.name}</option>;
+        });
+
+        return (
+            <div className="c-search-language__select-wrapper">
```

Comment:
This is not proper BEM usage
```suggestion
            <div className="c-search__language-wrapper">
```